docker/enrollment-server.xml

```diff
@@ -40,8 +40,7 @@
     <Parameter name="enrollment-server.activation-spawn.enabled" value="${ENROLLMENT_SERVER_ACTIVATION_SPAWN_ENABLED}"/>
 
     <!--  User-info configuration  -->
-    <Parameter name="enrollment-server.user-info.minimal-provider.enabled" value="${ENROLLMENT_SERVER_USER_INFO_MINIMAL_ENABLED:false}"/>
-    <Parameter name="enrollment-server.user-info.rest-provider.enabled" value="${ENROLLMENT_SERVER_USER_INFO_REST_ENABLED:false}"/>
+    <Parameter name="enrollment-server.user-info.provider" value="${ENROLLMENT_SERVER_USER_INFO_PROVIDER:}"/>
```

Comment:
Last time the value without default crash the app during the startup. Please test it or put a default here.

Reply:
I would like to see the error message. This is legit and working. It means no user-info provider at all. I have tested that. `${ENROLLMENT_SERVER_USER_INFO_PROVIDER:}` has a default value, which is empty.